instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -83,21 +130,25 @@ class FalconInstrumentor(BaseInstrumentor):
 
     def _instrument(self, **kwargs):
         self._original_falcon_api = falcon.API
-        falcon.API = _InstrumentedFalconAPI
+        falcon.API = partial(_InstrumentedFalconAPI, **kwargs)
```

Comment:
Should we not have explicit `kwargs` names like the other instrumentations?

Reply:
Do you mean something like:

```python
        falcon.API = partial(
            _InstrumentedFalconAPI,
            traced_request_attributes=kwargs.pop("traced_request_attributes", None),
            request_hook=kwargs.pop("request_hook", None),
            response_hook=kwargs.pop("response_hook", None),
        )
```
?

We could but we have to do the same thing again below so don't see a lot of value in it. Still open to adding it.

Reply:
Not a strong opinion.